support/aws-kinesis-agent.cron

```diff
@@ -1 +1 @@
-* * * * * root /usr/bin/aws-kinesis-agent-babysit > /dev/null 2>&1
+* * * * * root service aws-kinesis-agent status > /dev/null 2>&1 || service aws-kinesis-agent start
```

Comment:
If we want to make the babysitter script less complex, we want to at least differentiate whether the agent was intent to be brought down or not